go/runtime/host/protocol/types.go

```diff
@@ -53,27 +55,27 @@ type Body struct {
 	Empty *Empty `json:",omitempty"`
 	Error *Error `json:",omitempty"`
 
-	// Worker interface.
-	WorkerInfoRequest                    *WorkerInfoRequest                    `json:",omitempty"`
-	WorkerInfoResponse                   *WorkerInfoResponse                   `json:",omitempty"`
-	WorkerPingRequest                    *Empty                                `json:",omitempty"`
-	WorkerShutdownRequest                *Empty                                `json:",omitempty"`
-	WorkerCapabilityTEERakInitRequest    *WorkerCapabilityTEERakInitRequest    `json:",omitempty"`
-	WorkerCapabilityTEERakInitResponse   *Empty                                `json:",omitempty"`
-	WorkerCapabilityTEERakReportRequest  *Empty                                `json:",omitempty"`
-	WorkerCapabilityTEERakReportResponse *WorkerCapabilityTEERakReportResponse `json:",omitempty"`
-	WorkerCapabilityTEERakAvrRequest     *WorkerCapabilityTEERakAvrRequest     `json:",omitempty"`
-	WorkerCapabilityTEERakAvrResponse    *Empty                                `json:",omitempty"`
-	WorkerRPCCallRequest                 *WorkerRPCCallRequest                 `json:",omitempty"`
-	WorkerRPCCallResponse                *WorkerRPCCallResponse                `json:",omitempty"`
-	WorkerLocalRPCCallRequest            *WorkerLocalRPCCallRequest            `json:",omitempty"`
-	WorkerLocalRPCCallResponse           *WorkerLocalRPCCallResponse           `json:",omitempty"`
-	WorkerCheckTxBatchRequest            *WorkerCheckTxBatchRequest            `json:",omitempty"`
-	WorkerCheckTxBatchResponse           *WorkerCheckTxBatchResponse           `json:",omitempty"`
-	WorkerExecuteTxBatchRequest          *WorkerExecuteTxBatchRequest          `json:",omitempty"`
-	WorkerExecuteTxBatchResponse         *WorkerExecuteTxBatchResponse         `json:",omitempty"`
-	WorkerAbortRequest                   *Empty                                `json:",omitempty"`
-	WorkerAbortResponse                  *Empty                                `json:",omitempty"`
+	// Runtime interface.
```

Comment:
Not sure about how this will work out on the rust side, but would this be cleaner if things were grouped in a `struct Runtime`?